README.md

```diff
@@ -129,25 +128,61 @@ testsys run aws-k8s \
   --ec2-provider-image "ec2-resource-agent:eks"
 ```
 
-## Development
+### Test Results
 
-### Project Structure
+Executing the `testsys run` command will kick off the setup and execution of
+the Sonobuoy tests on an EKS cluster using Bottlerocket as the host OS for
+the cluster nodes.
 
-- `model` is the root dependency.
-It includes the CRDs and clients for interacting with them.
+To get the status and results of test execution, run the `testsys status`
+command for a high level summary:
 
-- `controller` contains the Kubernetes controller responsible for running resource and test pods.
+```shell
+ NAME           TYPE   STATE    PASSED   SKIPPED   FAILED
+ testsys-demo   Test   passed   1        5772      0
+```
 
-- `agent` contains libraries with the traits and harnesses for creating test and resource agents.
+When the test run has completed you may get the full logs of the test
```

Comment:
I'm not sure I understand this comment. Can you elaborate?

Reply:
^ means have a line per sentence, i. e.:

```
This is my large sentence.
This is my other sentence.
```

instead of:

```
This is my large sentence. This is my other sentence.
```

Reply:
Done